lib/ecto/repo.ex

```diff
@@ -1376,9 +1381,9 @@ defmodule Ecto.Repo do
 
       import Ecto.Changeset, only: [change: 2]
 
-      MyRepo.transaction(fn ->
-        MyRepo.update!(change(alice, balance: alice.balance - 10))
-        MyRepo.update!(change(bob, balance: bob.balance + 10))
+      MyRepo.transaction(fn repo ->
```

Comment:
I'm not sure about this: Edit the existing example or add a new one or just skip an example for the new option?

Reply:
Let's show two separate examples!